lib/ansible/modules/system/reset_ssh_session.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
```

Comment:
However, modules which are only documentation (as this is), should not have a shebang. Here's an example of existing "module" which is documentation only:

https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/inventory/add_host.py